client/stylesheets/components/_generic.scss

```diff
@@ -14,7 +14,7 @@
 
 .full.screen {
   width: 100vw;
-  height: 80vh;
+  height: 100%;
```

Comment:
That's why it will break, we need to specify more height overthere to span 88 of the view port as min-height to keep a (Full screen like) effect